tests/test_class_sh_property.py

```diff
@@ -9,23 +9,19 @@
 
 @pytest.mark.xfail("env.PYPY", reason="gc after `del field` is apparently deferred")
 @pytest.mark.parametrize("m_attr", ["m_valu_readonly", "m_valu_readwrite"])
-def test_valu_getter(msg, m_attr):
+def test_valu_getter(m_attr):
```

Comment:
nit: this function name is apparently misspelled

Reply:
It's intentional, see comment at top of this file, which points here:

https://github.com/pybind/pybind11/blob/feed8b15b581b317c92f00777c15d8ed5fe8a56f/tests/test_class_sh_property.cpp#L1-L4

(The original idea was actually to make [this code in test_class_sh_basic.cpp](https://github.com/pybind/pybind11/blob/feed8b15b581b317c92f00777c15d8ed5fe8a56f/tests/test_class_sh_basic.cpp#L70-L81) more readable.)